manifests/params.pp

```diff
@@ -30,8 +30,7 @@
   $mmonit_password           = 'monit'
   $mmonit_without_credential = false
 
-  # <OS family handling>
-  case $::osfamily {
+  case $facts['osfamily'] {
```

Comment:
that's still a legacy fact
```suggestion
  case $facts['os']['family'] {
```

Reply:
see https://github.com/voxpupuli/puppet-monit/pull/24#issuecomment-1941505573